upgrade-2.x.sh

```diff
@@ -249,6 +250,176 @@ function device_type_match {
     echo "${match}"
 }
 
+# Pre update cleanup: remove some not-required files from the boot partition to clear some space
+function pre_update_pi_bootfiles_removal {
+    local boot_files_for_removal=('start_db.elf' 'fixup_db.dat')
+    for f in "${boot_files_for_removal[@]}"; do
+        echo "Removing $f from boot partition"
+        rm -f "/mnt/boot/$f"
+    done
+    sync
```

Comment:
You can sync just the boot partition. `sync /mnt/boot`

Reply:
Done